lib/dug/runner.rb

```diff
@@ -33,6 +33,7 @@ def unprocessed_notifications(use_cache: true)
         @unprocessed_notifications = nil
       end
       unprocessed_label = servicer.labels(use_cache: use_cache)[Dug.configuration.unprocessed_label_name]
+      raise "Create the label '#{Dug.configuration.unprocessed_label_name}' on gmail" unless unprocessed_label
```

Comment:
I ran into this as well the first time I ran the script (`script.rb`.) I even thought I followed @chrisarcand's detailed instructions.  

In my case I set up `UnProcessed` in camel case. 

It would be a nice fix to have.